demos/prefsServerIntegration/css/style.css

```diff
@@ -0,0 +1,4 @@
+/* In the preferences server integration demo, show the selfVoicingToggle button on the congratulations page */
+.gpii-fd-lastPanel.gpii-fd-prefsServerIntegration .gpii-fd-selfVoicingToggle {
```

Comment:
After meeting with Dana, will show "Turn voice on/off" button on the confirmation page of all demos.